docs/DeveloperGuide.md

```diff
@@ -711,48 +806,49 @@ should end at the destroy marker (X) but due to a limitation of PlantUML, the li
-| `* * *`  | tutor with many persons in the contact book | sort persons by name                          | locate a person easily                                                   |
-| `* * *`  | responsible tutor                           | track students' progress on their assignments | identify and reach out to those who need help                            |
-| `* * *`  | tutor                                       | list all students I am teaching               | ensure I added right and correct number of people before                 |
-| `* * `   | TA                                          | assign a similar task to all students at once | save time manually assigning one by one                                  |
-| `* * `   | TA                                          | remove all assignments that are completed     | save time manually removing each assignment one by one                   |
-| `* *`    | busy tutor                                  | list certain people I interacted frequently   | save time searching their name whenever I start app                      | 
-| `*`      | clumsy tutor                                | undo actions                                  | recover information that I accidentally delete                           |
+| Priority | As a …​                                 | I want to …​                                   | So that I can…​                                                           |                                                 
+| -------- | ------------------------------------------ | ------------------------------------------------- | ---------------------------------------------------------------------------- |
+| `* * *`  | TA using TA<sup>2</sup> for the first time | see all commands available                        | recall commands and use them properly when I forget how to use the app       |
+| `* * *`  | TA                                         | add information of a student                      | -                                                                            |
+| `* * *`  | TA                                         | delete a student                                  | remove student entries that I no longer need                                 |
+| `* * *`  | TA                                         | find a person by name or module                   | locate details of persons without having to go through the entire list       |
+| `* * *`  | TA                                         | assign tasks to students                          | -                                                                            |
+| `* * *`  | TA                                         | delete tasks assigned before                      | remove the outdated assignments for students                                 |
```

Comment:
remove `obsolete` assignments....


Should just be very general, instead of defining exactly why you want to delete it